spec/controller/books_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe BooksController, type: :controller do
+  describe "GET #show" do
+    let!(:book) {FactoryBot.create :book}
+
+    it "returns a success response" do
+      get :show, params: {id: book}
+      expect(response).to be_successful
+    end
+
+    it "assigns @book when book exist" do
+      get :show, params: {id: book}
+      expect(assigns(:book)).to eq book
+    end
```

Comment:
2 cái `it` ni gộp vào 1 `context "when book exist"` nè

dùng before để load cho action luôn nè